### DISCUSSION TITLE
Yaru variant how to

### DISCUSSION THREAD

**Discussion by meetdilip** [ubuntu/yaru#3747] [Q&A]
Hi, I would like to create a variant of Yaru and use it under another name. At the moment, it is just a personal project with a few colour changes to existing icons. Target OS is Ubuntu 22.04.

If I download Yaru-master and make necessary changes to the SVGs, can you please tell me how to build them into an icon theme with a different name, say `Modified`? 

I am trying to use it as a local icon theme under `.icons` folder. So, I guess the installation part is not required. 

Thanks.

**Comment by Jupi007** [comment]
Hi @meetdilip,
honestly I have no idea of how you could do this.

@3v1n0 added support for accents, but I didn't dived into the details of how to.
Maybe he could explain you?

**Comment by meetdilip** [comment]
Thanks for the response. If I am to change the folder icon, I see that there are many sizes in a single file. Can I draw the big one ( differently ) and simply resize and place it to match the smaller ones?

**Reply by Jupi007** [reply]
Well, you shouldn't because each size is designed to be pixel perfect (aligned on the grid).
So you can, but it will looks blurry :)

**Reply by meetdilip** [reply]
What is the right way to replace an icon for all sizes in the SVG?

**Reply by Jupi007** [reply]
Well, redraw it at each sizes :)
Largest ones are more detailed, and the smallest are very much simple.

**Reply by meetdilip** [reply]
I agree. But how to place it correctly on all sizes? Just match the grid lines in Inkscape?

**Reply by meetdilip** [reply]
Consider the default folder as an example please.

**Reply by Jupi007** [reply]
Oh, well you have a hidden `baseplate` layer that you can show up in Inkscape.
Just remember to hide it before render the icons :)

**Reply by meetdilip** [reply]
Awesome!! Thanks :)

**Reply by Jupi007** [reply]
You're welcome :)

**Comment by ojovial** [comment]
I will be nice to have this how-to 
I understand how to generate png icons theme from the inscape files but i would like to create new accent color for gtk themes.
There is  a file create-new-variant.md , but I've never been able to do it with the instructions provided.
So I stay limited to create new-variant.
If you have any way to ask help or support, thank you.
